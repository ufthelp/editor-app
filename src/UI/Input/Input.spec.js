import React from 'react';
import {shallow} from 'enzyme';
import Input from './Input';


describe('<Input /> Component', ()=>{
    let wrapper;
    beforeEach(()=> wrapper = shallow( <Input 
        iconImage='smile'
    />))

    it('should render correctly', () => expect(wrapper).toMatchSnapshot());

    it('should render the value of title prop', () => {
        wrapper.setProps({ title: 'file' });
        expect(wrapper.text()).toEqual('file');
    });
})

