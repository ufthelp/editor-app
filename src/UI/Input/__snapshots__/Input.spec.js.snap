// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> Component should render correctly 1`] = `ShallowWrapper {}`;
